dune_api_scripts/queries/parsed_app_data.sql

```diff
@@ -12,32 +13,36 @@ AS (
             app_data as app_id,
             referrer::json -> 'appCode' as app_code,
             referrer::json -> 'version' as app_version,
-            (referrer::json -> 'metadata')::json as metadata
+            -- Notice there are 2 different environment fields.
+            -- One of them being utilized more than the other
+            referrer::json -> 'environment' as backend_env,
+            (referrer::json -> 'metadata')::json -> 'environment' as meta_env,
+            ((referrer::json -> 'metadata')::json -> 'referrer')::json -> 'address' as referrer,
+            ((referrer::json -> 'metadata')::json -> 'referrer')::json -> 'version' as referral_version,
```

Comment:
Mostly nitpicking, but in my humble opinion we should keep `referrer_version` instead of `referral_version` to keep the reference to the origin as similar as possible.

Reply:
Yea sure we can do this.